apollo-router/src/executable.rs

```diff
@@ -205,18 +205,14 @@ pub struct Opt {
     #[clap(skip = std::env::var("APOLLO_GRAPH_REF").ok())]
     apollo_graph_ref: Option<String>,
 
-    /// Your Apollo Router entitlement.
+    /// Your Apollo Router license.
     /// EXPERIMENTAL and not subject to semver.
-    #[clap(skip = std::env::var("APOLLO_ROUTER_ENTITLEMENT").ok())]
-    apollo_router_entitlement: Option<String>,
+    #[clap(skip = std::env::var("APOLLO_ROUTER_LICENSE").ok() || std::env::var("APOLLO_ROUTER_ENTITLEMENT").ok())]
```

Comment:
This fails to compile because `Result::ok` returns an `Option`, not a boolean. Untested fix:

```suggestion
    #[clap(skip = std::env::var("APOLLO_ROUTER_LICENSE").or_else(|| std::env::var("APOLLO_ROUTER_ENTITLEMENT")))]
```